packages/core/src/Cardano/types/Transaction.ts

```diff
@@ -11,6 +11,7 @@ import { PlutusData } from './PlutusData';
 import { ProposalProcedure, VotingProcedures } from './Governance';
 import { RewardAccount } from '../Address';
 import { Script } from './Script';
+import { Serialization } from '../..';
```

Comment:
This might cause circular import issues. Anyway I think a better place for the new `isTxBodyWithHash` util is in `wallet` package because that where the `Serialization.TxCBOR | TxBodyWithHash` is used
```suggestion
import type { Serialization } from '../..';
```

Reply:
Moved the implementation in the wallet package: 84396c6